### DISCUSSION TITLE
Changing web server default 8080 port

### DISCUSSION THREAD

**Discussion by dmarques25** [DefectDojo/django-DefectDojo#6087] [Q&A]
Hi,

I'm new to DefectDojo and trying to make some testing. I was able to install it on a testing machine to test, but now I need start working on it on my production environment, but I've got already something else running on port 8080, so I need to move DefectDojo to another port. 
I've changed the docker-compose.yml file, changing the port from 8080 to 8081, but now the application doesn't start.
Is there anything else I need to change in order to change the default port?

Thanks

**Comment by mtesauro** [comment]
Generally, changing ports in compose doesn't cause a problem.  FWIW, I tend to setup DefectDojo similar to [this](https://github.com/DefectDojo/Community-Contribs/tree/master/CentOS-7-deploy) with a compose like [this](https://github.com/DefectDojo/Community-Contribs/blob/master/CentOS-7-deploy/dojo/docker-compose.yml).

Did you change the target or published port in compose?

By "application doesn't start", I'm assuming `docker-compose up` failed.  If that's correct, what errors were shown when running that command?

If compose comes up cleanly, what are the logs from nginx and/or uwsgi/dojo?
e.g. docker logs defectdojo_nginx
